src/components/Button.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ buttonName = 'Button', color, wide }) => {
+  let theClass = 'calculator__button';
+  if (color) {
+    theClass = 'calculator__button button-gray';
+  }
+  if (wide) {
+    theClass = 'calculator__button button-gray zero';
+  }
+  return (
+    <button
+      type="button"
+      className={theClass}
+    >
+      {buttonName}
+    </button>
+  );
+};
+
+Button.propTypes = {
+  buttonName: PropTypes.string,
+  color: PropTypes.bool,
```

Comment:
`color` should be a string instead and defualt `color` for button is orange ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png))